src/Item_Devices.php

```diff
@@ -684,16 +686,18 @@ public static function showForItem(CommonGLPI $item, $withtemplate = 0)
                 $table_options['itemtype'] = $itemtype;
                 $link                      = getItemForItemtype(static::getType());
 
-                $link->getTableGroup(
-                    $item,
-                    $table,
-                    $table_options,
-                    $delete_all_column,
-                    $common_column,
-                    $specific_column,
-                    $delete_column,
-                    $dynamic_column
-                );
+                if ($link instanceof Item_Devices) {
```

Comment:
Maybe not needed anymore with the previous recommendation of not using `getType`.